source/patterns/@aws-solutions-constructs/aws-constructs-factories/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "@aws-solutions-constructs/aws-constructs-factories",
+  "version": "0.0.0",
+  "description": "Factories to allow creation of individual resources",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/awslabs/aws-solutions-constructs.git",
+    "directory": "source/patterns/@aws-solutions-constructs/aws-dynamodbstreams-lambda"
```

Comment:
this directory looks wrong "aws-dynamodbstreams-lambda"

Reply:
Addressed